kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java

```diff
@@ -177,6 +178,28 @@ public void testDecisionExpressionLiteral() throws Exception {
                                   "/dmn:text[text()='123']");
     }
 
+    @Test
+    public void testBendPoints() throws Exception {
+        final String expected = loadResource("bend-points.xml");
+        setContent(expected);
+
+        final String actual = getContent();
+        assertThat(actual).isNotBlank();
+
+        final List<String> edges = Arrays.asList("dmnedge-drg-_5FF07784-BB77-4632-985F-2805302C03D3-AUTO-TARGET",
+                                                 "dmnedge-drg-_99B590D4-A82B-43D9-9DD7-A3A4D57D8CA5-AUTO-SOURCE-AUTO-TARGET",
+                                                 "dmnedge-drg-_2B636F4A-3A7F-4C38-9D42-B000D1E0CCF2-AUTO-SOURCE-AUTO-TARGET");
+
+        edges.forEach(edge ->
+                              XmlAssert.assertThat(actual)
+                                      .withNamespaceContext(NAMESPACES)
+                                      .hasXPath("/dmn:definitions/dmndi:DMNDI/dmndi:DMNDiagram" +
+                                                        "/dmndi:DMNEdge[@id='" + edge + "']" +
+                                                        "/di:waypoint")
+                                      .hasSize(3)
+        );
+    }
+
```

Comment:
Hi @jomarko, this test looks to me like it can be a normal java unit test. Is Selenium has any benefits there? Selenium tests are expensive to execute and maintain, so I think it should be a reason to use.

Reply:
@hasys I would like to check remarshalling of diagram (I was told by @manstis to do it in this was historically), please point me to a non selenium tests, that does such a thing for a DMN, then I can rewrite this test to be a non selenium.

I agree selenium tests are more expensive than non selenium ones, but I need a guidance to rewrite it or keep it as it is.

Reply:
Good point, @jomarko, on client side it's currently can be complicated.

Reply:
@hasys do you agree to proceed with this PR as it is?

Reply:
Sure, go ahead. Thank you!

Reply:
thank you